### ISSUE TITLE
Dependency injection causing this module to not work at all

### ISSUE THREAD

**Issue by hulkish** [ui-router/visualizer#3]
I'm seeing dependency injection errors while trying to use this module:
```
angular.js:13708 Error: [$injector:strictdi] function($transitions, $timeout, d3ng, easing, uirTransitionsViewConfig) is not using explicit annotation and cannot be invoked in strict mode
```

 After digging into the release/visualizer.js source I see this (amungst others):
```
statevis_module_1.app.run(function ($rootScope, $injector, uirStateVisService) {
```

Shouldn't this look like this?
```
statevis_module_1.app.run(['$rootScope', '$injector', 'uirStateVisService', function ($rootScope, $injector, uirStateVisService) {
```

**Comment by hulkish** [comment]
Correction... this is only a problem when using strictDi... like so:

```js
angular.bootstrap(document, ['app'], {
    strictDi: true
});
```

That said, I'm unfamiliar with typescript - so im not really sure what is being done to address this.

**Comment by christopherthielen** [comment]
@hulkish closing because we released version 2.0.0 and it has a totally different way to bootstrap.

```js
app.run(function (ng1UIRouter) {
  // or var vis = require('ui-router-visualizer')
  // or import * as vis from 'ui-router-visualizer';
  var vis = window['ui-router-visualizer'];

  vis.visualizer(ng1UIRouter);
});
```
